test/common/source/common.d

```diff
@@ -0,0 +1,45 @@
+module common;
+
+/// Name of the log file
+enum logFile = "test.log";
+
+/// has true if some test fails
+bool any_errors = false;
+
+/// prints (non error) message to standard output and log file
+void log(Args...)(Args args)
+	if (Args.length)
+{
+	import std.conv : text;
+	import std.stdio : File, stdout;
+
+	const str = text("[INFO] ", args);
+	version(Windows) stdout.writeln(str);
+	else stdout.writeln("\033[0;33m", str, "\033[0m");
+	stdout.flush;
+	File(logFile, "a").writeln(str);
+}
+
+/// prints error message to standard error stream and log file
+/// and set any_errors var to true value to indicate that some
+/// test fails
+void logError(Args...)(Args args)
+{
+	import std.conv : text;
+	import std.stdio : File, stderr;
```

Comment:
Move those to module level, no point in using function-local imports when all of them import the same thing.

Reply:
I write PoCs very frequently. It's much easier to refactor the code when imports are local and that's my habit now. But I agree, here it is useless.